ff_meters_LevelMeterSource.h

```diff
@@ -102,12 +102,12 @@ class LevelMeterSource
         float getAvgRMS () const
         {
             if (rmsHistory.size() > 0)
-                return std::sqrt (std::accumulate (rmsHistory.begin(), rmsHistory.end(), 0.0) / rmsHistory.size());
+                return std::sqrt (std::accumulate (rmsHistory.begin(), rmsHistory.end(), 0.0f) / rmsHistory.size());
 
-            return std::sqrt (rmsSum);
+            return (float) std::sqrt (rmsSum);
         }
 
-        void setLevels (const juce::int64 time, const float newMax, const float newRms, const juce::int64 holdMSecs)
+        void setLevels (const juce::int64 time, const float newMax, const float newRms, const juce::int64 _holdMSecs)
```

Comment:
Can you please remove the underscore?
I follow the style not to use them, since they might be used internally by some libraries afaik.
The name should be fine, since the member holdMSecs is defined in the parent class... If not use something different like `newHold` or `holdMSecsToUse`